fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceTokenValueRec.java

```diff
@@ -31,28 +37,34 @@
      * @param logicalResourceId
      * @param externalSystemName
      * @param externalRefValue
+     * @param compositeId
+     * @param systemLevel
      */
     public ResourceTokenValueRec(int parameterNameId, String resourceType, long resourceTypeId, long logicalResourceId,
-        String codeSystem, String externalRefValue) {
-        this(parameterNameId, resourceType, resourceTypeId, logicalResourceId, codeSystem, externalRefValue, null);
+        String codeSystem, String externalRefValue, Integer compositeId, boolean systemLevel) {
+        this(parameterNameId, resourceType, resourceTypeId, logicalResourceId, codeSystem, externalRefValue, null, compositeId, systemLevel);
     }
 
     /**
-     * Public constructor. For use to create a versioned resource reference
+     * Public constructor. Used to create a versioned resource reference
      * @param parameterNameId
      * @param resourceType
      * @param resourceTypeId
      * @param logicalResourceId
      * @param externalSystemName
      * @param externalRefValue
      * @param refVersionId
+     * @param compositeId
+     * @param systemLevel
      */
     public ResourceTokenValueRec(int parameterNameId, String resourceType, long resourceTypeId, long logicalResourceId,
```

Comment:
Not related to this changeset, but just a general question:  what is the "Rec" part of this name short for?

Reply:
record - a logical entry in the database.